pandas/tests/arrays/sparse/test_array.py

```diff
@@ -215,6 +215,7 @@ def test_scalar_with_index_infer_dtype(self, scalar, dtype):
         assert exp.dtype == dtype
 
     @pytest.mark.parametrize("fill", [1, np.nan, 0])
+    @pytest.mark.filterwarnings("ignore:Sparse:FutureWarning")
```

Comment:
I think you don't need these as a prior PR added this to setup.cfg

Reply:
The setup.cfg has an `error:::` config to elevate unhandled warnings to errors. We still need these otherwise the tests would fail.

We have a single test asserting that `SparseSeries.__init__` warns, and explicitly ignore the warnings elsewhere.